### ISSUE TITLE
Investigates Flutter voice access support for Android

### ISSUE THREAD

**Issue by chunhtai** [flutter/flutter#76827]
We need to make sure flutter is accessible when uses android voice access

Action items:

- [x] Figure out why widget only shows numeric label when the user speak "show labels"
- [x] Entering text does not work. https://github.com/flutter/flutter/issues/77271
- [x] deleting text does not work. https://github.com/flutter/flutter/issues/77965

**Comment by chunhtai** [comment]
some updates:

The `show labels` command does not show the string label, but only shows the numeric label. Not entirely sure what are the expectations but we should figure out why it does not show the string label.

Text editing does not work
